contracts/supplementary/rebalancing-manager/BTCETHRebalancingManager.sol

```diff
@@ -109,28 +106,32 @@ contract BTCETHRebalancingManager {
         address _setTokenFactory,
         address _auctionLibrary,
         uint256 _auctionTimeToPivot,
-        uint256 _btcMultiplier,
-        uint256 _ethMultiplier
+        uint256[] _multipliers,
```

Comment:
Would we want to fix this to `uint[2]` and the same w/ allocationBounds?

Reply:
Random thought. Can't we have a uint256[4] instead of 2 uint256[2]?

Also, consider doing uint64[4] or uint128[2] to pack the numbers together, since we don't expect multipliers or lower/upper bound to be very huge.